documentation added for pipelines (#2793)
weaveworks/weave-gitops: website/docs/enterprise/pipelines/spec/v1alpha1/pipeline.mdx

---
title: Pipeline
sidebar_position: 1
hide_title: true
---
# Pipeline <TierLabel tiers="enterprise" />

The Pipeline API defines a resource for continuous delivery pipelines.

> An example of fully defined pipeline could be found below.

Reviewer: ```suggestion
An example of a fully defined pipeline is shown below.
```
